### DISCUSSION TITLE
Dokku deploy

### DISCUSSION THREAD

**Discussion by corentinclichy** [documenso/documenso#345] [Ideas]
Hey team, 

I saw there is railway and render one click button. 

Could be really nice to have a Dokku plugin/way to deploy a documenso in a dokku server !

Have a nice day!

Corentin

**Comment by Mythie** [comment]
It certainly would be, we do plan on expanding our set of 1 click deploys over time 😸

**Reply by corentinclichy** [reply]
Nice ! Thanks a lot for the good work ! I really enjoy the experience so far
